config/config.go

```diff
@@ -0,0 +1,63 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"code.cloudfoundry.org/lager"
+)
+
+var (
+	logLevels = map[string]lager.LogLevel{
+		"DEBUG": lager.DEBUG,
+		"INFO":  lager.INFO,
+		"ERROR": lager.ERROR,
+		"FATAL": lager.FATAL,
+	}
+)
+
+type Config struct {
+	LogLevel      lager.LogLevel
+	BrokerAPIHost string
+	BrokerAPIPort string
```

Comment:
Renamed to ListenPort. The host can be removed as per your comment regarding `http.ListenAndServe` having a useful default value.